src/main/java/chipchat/action/AddTask.java

```diff
@@ -0,0 +1,48 @@
+package chipchat.action;
+
+import chipchat.App;
+import chipchat.exception.ChipchatException;
+import chipchat.storage.Storage;
+import chipchat.task.Task;
+import chipchat.task.Event;
+import chipchat.task.Deadline;
+import chipchat.task.Todo;
+import chipchat.task.TaskList;
+import chipchat.ui.Ui;
+
+import java.time.LocalDate;
+
+public class AddTask extends Action {
```

Comment:
Perhaps can change the name to sound more "noun" instead. It sounds more verbish right now 😆. Can check here!
https://nus-cs2103-ay2324s2.github.io/website/schedule/week4/topics.html#w4-6-code-quality-naming